### DISCUSSION TITLE
번역에 참여하고싶습니다.

### DISCUSSION THREAD

**Discussion by bamjun** [rust-kr/doc.rust-kr.org#19] [General]
저두 참여 가능할까요 ?

**Comment by rinthel** [comment]
안녕하세요~ 번역 참여에 관심 가져주셔서 감사합니다!
타이밍이 좀 아쉽게 되었는데, 출판사의 지원을 받아 2021판 번역을 제가 주도적으로 진행하게 되었습니다.
초벌 번역이 끝난 이후가 되면 그때는 다시 공동 참여가 가능한 형태로 해보겠습니다~
아, 초벌 번역이 되면 그냥 출판만 되는 것은 아니고, github에도 함께 업데이트가 될 예정입니다!

**Comment by JAICHANGPARK** [comment]
완료되시면 향후 저도 참여하고 싶네요. 
대기 걸어두겠습니다 😄
